app/src/main/java/com/example/notes2text/file_sharing/adapters/ThirdPartySender.java

```diff
@@ -0,0 +1,47 @@
+package com.example.notes2text.file_sharing.adapters;
+
+import com.example.notes2text.file_sharing.use_case.ThirdPartyOutputModel;
+import com.example.notes2text.file_sharing.use_case.ThirdPartyOutputBoundary;
+
+import java.util.ArrayList;
+
+import android.net.Uri;
+import android.content.Intent;
+
+
+public class ThirdPartySender implements ThirdPartyOutputBoundary {
+
+
+    /**
+     * Using the list of content Uri in the given outputModel, create a new Intent
+     * to access the share sheet function. Only sends text file of any type. OutputModel
+     * is returned so that context may be used to check the results of the action.
```

Comment:
This method does not return output model - if this is intentional, please change your docstring to reflect that the output model is not returned.